easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.1.3-PGI-19.5-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
```

Comment:
@JackPerdue This could/should be cleaned up since we have a custom easyblock for `OpenMPI` now, see https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.1-GCC-8.3.0-2.32.eb